src/composables/map/ol.composable.ts

```diff
@@ -1,27 +1,73 @@
 import type BaseLayer from 'ol/layer/Base'
 import ImageLayer from 'ol/layer/Image'
-import type TileLayer from 'ol/layer/Tile'
+import TileLayer from 'ol/layer/Tile'
 import type OlMap from 'ol/Map'
 import { ImageWMS, WMTS } from 'ol/source'
+import WmtsTileGrid from 'ol/tilegrid/WMTS'
+import { getTopLeft } from 'ol/extent.js'
+import { get as getProjection, transformExtent } from 'ol/proj'
 
-import { createBgWmtsLayer } from '@/composables/background-layer/background-layer.wmts-helper'
 import { layersCache } from '@/stores/layers.cache'
 import type { Layer, LayerId } from '@/stores/map.store.model'
+import { PROJECTION_DESTINATION, PROJECTION_SOURCE } from './map.composable'
+import { isHiDpi, stringToBoolean } from '@/services/utils'
+import { storageHelper } from '@/services/state-persistor/storage/storage.helper'
+import { SP_KEY_IPV6 } from '@/services/state-persistor/state-persistor.model'
 
 const proxyWmsUrl = 'https://map.geoportail.lu/ogcproxywms'
 export const remoteProxyWms = 'https://map.geoportail.lu/httpsproxy'
 
+const TILE_GRID_RESOLUTION = [
```

Comment:
would it be possible to move the constant definitions into a config file ?

Reply:
Yes, moved into wmts fixture file.